website/docs/r/tg_connection_actions.html.markdown

```diff
@@ -0,0 +1,29 @@
+---
+subcategory: "Transit Gateway"
+layout: "ibm"
+page_title: "IBM : tg_connection_actions"
+description: |-
+  Perform actions on a connection for a Transit Gateway
+---
+
+# ibm_tg_connection_action
```

Comment:
make sure this name is as same as resource name

page title says `tg_connection_actions` but this says `_action`

Reply:
I renamed the resource to '_action', so this is correct